### DISCUSSION TITLE
Можно было бы синхронизировать только цену и кол-во товара

### DISCUSSION THREAD

**Discussion by CapitalistFox** [wpcraft-ru/wooms#513] [Ideas]
Я понимаю, что прошу дофига, но всё же.
Было бы неплохо если бы:
- Можно было бы синхронизировать только цену и кол-во товара (без синхронизации категорий и прочего и без автоматического создания товаров). Поскольку на одном аккаунте МС может быть несколько сайтов с общим складом но разным ассортиментом.

**Comment by aiiddqd** [comment]
думаю технически это реально, но как мне кажется это надо писать отдельный доп плагин - что проще. Думаю можно написать за 2-3 дня при наличии навыков программирования в этой теме.

Если дописывать базовый плагин - там вероятно придется много всего переписывать и это может быть очень дорого.

**Reply by varikweru** [reply]
Я у себя сделал доп.поле "Не грузить на сайт" (для товаров, которые продаются только на маркетплейсах) и прямо в функцию **product_update** в файле **Products.php** вставил проверку типа
```
	$not_add = false;
	$attr_array = $row['attributes'];
        $attr_id = 'id доп.поля';
	foreach ($attr_array as $attribute) {
	    if ($attribute['id'] == $attr_id && $attribute['value'] == 'true') {
	        $not_add = true;
	        break;
	    }
	}

	if ( empty( intval( $product_id ) ) && $not_add == false) {
		$product_id = add_product( $row );
	}
```
Знаю, что это максимально грубое решение, но в целом, мне кажется, просто добавить в качестве опции название подобного поля и сделать такую проверку в файле **Products.php** не так сложно. В таком случае даже ситуация, как у CapitalistFox будет решаемой.
